src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -13,6 +13,40 @@ const MISSING_BLOCK_ERROR_MESSAGE = 'The "block" field type needs at least one t
 const BLOCK_PREVIEW_TAG = 'sulu.block_preview';
 
 export default class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
+    componentDidUpdate(prevProps: FieldTypeProps<Array<BlockEntry>>) {
+        const {defaultType, types, value: value, onChange} = this.props;
```

Comment:
Could be written as just `value`, not `value: value`.

Reply:
Maybe we could even include the [eslint rule for that](https://eslint.org/docs/rules/no-useless-rename)?

Reply:
oh, and please sort it alphabetically :slightly_smiling_face:

Reply:
rule added and sorted alphabetically :)